abpframework/abp
# Add Custom Component Feature to ExtensibleForm.

Resolves #13785

Test Preparation
 - go to npm>ng_packs and run `yarn run build:all`
- run backend with extra-properties 
- run ng-packs/dev-app. 

Test steps
- [x] my-profile page re-written with extensible form. So it should work as before.  All functionality that in my-profile should be tested.
- [x] Add extra properties  on Backend (https://docs.abp.io/en/abp/latest/Module-Entity-Extensions)
- [x] The extra properties should visible and changeable on the profile page. 


the task related to https://github.com/abpframework/abp/pull/13700
